src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx

```diff
@@ -11,13 +11,13 @@ redirects:
   - /docs/onelogin-scimsso-application-configuration
 ---
 
-import oneloginRulesConditionsActions from 'images/onelogin_rules_conditions_actions.png'
+import accountsOneLoginRulesConditionsActions from 'images/accounts_screenshot-crop_OneLogin-rules-conditions-actions.png'
 
-import oneloginProvisioningForm from 'images/onelogin_provisioning_form.png'
+import accountsOneLoginProvisioningForm from 'images/accounts_screenshot-crop_OneLogin-provisioning-form.png'
 
-import oneloginParameterFormGeneral from 'images/onelogin_parameter_form_general.png'
+import accountsOneLoginGeneralParameterForm from 'images/accounts_screenshot-crop_OneLogin-general-parameter-form-.png'
 
-import oneloginParameterFormSpecific from 'images/onelogin_parameter_form_specific.png'
```

Comment:
@zuluecho9 looks like the image filename wasn't updated in this import statement